RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -250,6 +250,46 @@ boolean LX_islandAccess()
 	return false;
 }
 
+boolean LX_lockPicking()
+{
+	if(!auto_have_skill($skill[Lock Picking]))
+	{
+		return false;
+	}
+
+	if(get_property("lockPicked").to_boolean() == true)
+	{
+		return false;
+	}
+
+	if(towerKeyCount(false) >= 3)
+	{
+		return false;
+	}
+
+	if(my_mp() < mp_cost($skill[Lock Picking]))
+	{
+		return false;
+	}
+
+	if(item_amount($item[Boris\'s Key]) == 0)
+	{
+		set_property("choiceAdventure1414", 1);
+	}
+	else if(item_amount($item[Jarlsberg\'s Key]) == 0)
+	{
+		set_property("choiceAdventure1414", 2);
+	}
+	else if(item_amount($item[Sneaky Pete\'s Key]) == 0)
+	{
+		set_property("choiceAdventure1414", 3);
+	}

```

Comment:
this should be in `auto_choice_adv.ash`
although this can also be moved there in a future PR, I don't mind

Reply:
I put it there initially but it wasn't called in response to using the skill

Reply:
Oh I just noticed. In that case I would add a comment "as of X does not work in choice adventure script"

Reply:
X can be date and / or mafia version